packages/calendar/test/utils/getFirstDayNextMonth.test.js

```diff
@@ -1,11 +1,17 @@
 import { expect } from '@open-wc/testing';
-import { formatDate } from '../../../localize/src/date/formatDate.js';
+import { isSameDate } from '../../src/utils/isSameDate.js';
 import { getFirstDayNextMonth } from '../../src/utils/getFirstDayNextMonth.js';
 
 describe('getFirstDayNextMonth', () => {
   it('returns the first day of the next month', () => {
-    expect(formatDate(getFirstDayNextMonth(new Date('2001/01/01')))).to.be.equal('01/02/2001');
-    expect(formatDate(getFirstDayNextMonth(new Date('2001/10/10')))).to.be.equal('01/11/2001');
-    expect(formatDate(getFirstDayNextMonth(new Date('2000/03/10')))).to.be.equal('01/04/2000');
+    expect(
+      isSameDate(getFirstDayNextMonth(new Date('2001/01/01')), new Date('2001/02/01')),
+    ).to.equal(true);
+    expect(
+      isSameDate(getFirstDayNextMonth(new Date('2001/10/10')), new Date('2001/11/01')),
+    ).to.equal(true);
+    expect(
+      isSameDate(getFirstDayNextMonth(new Date('2000/03/10')), new Date('2000/04/01')),
+    ).to.equal(true);
```

Comment:
these tests failed because of a different order of execution of tests and because they depended on the page locale

I refactored them to be locale independent, so now they work